### ISSUE TITLE
Test set output summaries for CNN/DM

### ISSUE THREAD

**Issue by alexgaskell10** [google-research/pegasus#22]
Hi Jingqing,

I am looking to run some analysis on PEGASUS' output summaries for CNN/DM. Can I get hold of these in any way? (I know I could just run the model to produce them myself but thought I would ask if they are already available before doing this!)

Great work on all this!

All the best,
Alex

**Comment by JingqingZ** [comment]
Hi Alex, I am afraid we can't share the output summaries because of some limitation. Please run the model and produce the outputs. Sorry for the inconvenience.

**Comment by alexgaskell10** [comment]
Thanks for coming back to me. Just trying to do this now and I have some follow up questions:
1. Is the model fine-tuned on CNN-DM available for download? 
2. If answer to above is no, how long does fine-tuning take?
3. How do I get the model to write output summaries?

Thanks!

**Comment by JingqingZ** [comment]
The fine-tuned models of all 12 datasets including CNN/DM are available. Please follow the README and download.

**Comment by alexgaskell10** [comment]
Thanks for your help so far. 

I have got everything set up and am running eval on the test set. It is running slow, even with GPU. It is taking 30s per sample and this doesn't fall if I increase batch size. Is this normal?

**Comment by JingqingZ** [comment]
I suspect the model may actually run on CPU. Please run `nvidia-smi` and check if the model uses GPU. The model should take at least 10000MB GPU memory. 

Others may be worth checking: 
* CUDA 10.0
* `tensorflow-gpu` is installed instead of `tensorflow`
* Check the printout log of the model and see if anything unexpected regarding GPU.

**Comment by alexgaskell10** [comment]
All looks to be using the GPU normally... Do you have an estimate for how long running an eval loop should take?

Nvidia-smi gives:
<img width="506" alt="Screenshot 2020-06-16 at 22 37 10" src="https://user-images.githubusercontent.com/51463426/84830961-5c2c3880-b022-11ea-9ace-900b3dd27fb9.png">

And the following logs on loading:
<img width="1380" alt="Screenshot 2020-06-16 at 22 36 45" src="https://user-images.githubusercontent.com/51463426/84830997-6ea67200-b022-11ea-8130-f77a4fc9e4cb.png">

**Comment by JingqingZ** [comment]
FYI, I used `batch_size=8` and it took around 10min for 100 CNN/DM examples on a single TITAN X.

**Comment by alexgaskell10** [comment]
Hmmm, weird how it's taking 5x as long for me as it did for you. Have had a bit of a look but nothing obvious I can see. I will have to park this now and move on.

**Comment by alexgaskell10** [comment]
Gave this one final try on a different GPU machine and it is working much better so am closing the issue. Thanks for your help!

**Comment by jacob-parnell-rozetta** [comment]
@alexgaskell10 out of curiosity, what were the specs of the new GPU machine you tested this on? I am running into a similar issue, and it takes quite a while to run an eval loop on CNN/DM, even increasing batch_size doesn't help.

**Comment by alexgaskell10** [comment]
So the slow version was using [Azure's NC6](https://docs.microsoft.com/en-us/azure/virtual-machines/nc-series) series (believe they are Tesla K80s) and the faster version was using GeForce GTX TITAN X. Both have 12Gb RAM. This made it much quicker but was still only half the speed Jingqing was reporting.
